.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: python
+
+sudo: false
+
+matrix:
+  include:
+    - python: 3.3
+      env:
+        - TASK=py33
+    - python: 3.4
```

Comment:
Not us. But since this is being open sourced I wanted to support Python 3 at least from 3.3.